packages/amplify_auth_cognito/example/lib/Widgets/ViewUserAttributes.dart

```diff
@@ -106,14 +106,14 @@ class _ViewUserAttributesState extends State<ViewUserAttributes> {
               child: ListView.builder(
                 itemCount: _userAttributes.length,
                 itemBuilder: (context, index) {
-                  var key = _userAttributes[index].userAttributeKey
-                      as CognitoUserAttributeKey;
-                  var value = _userAttributes[index].value;
-                  var stringValue = value.toString();
+                  AuthUserAttribute atrribute = _userAttributes[index];
+                  CognitoUserAttributeKey userAttributeKey =
+                      atrribute.userAttributeKey as CognitoUserAttributeKey;
+                  String value = atrribute.value;
                   return ListTile(
-                    title: Text(key.key),
```

Comment:
I renamed `key` to `userAttributeKey` because `key.key` was confusing me 🙂 

This is well out of scope of this change, but `CognitoUserAttributeKey` is new in 0.3.0. The definition is below. I think `key` should be renamed to `name` or `keyName` because the entire thing is the key, and this field is the name of that key.

If we believe this API makes more sense, it would be good to update now, because it will be a breaking change if done after 0.3.0. I could include this.

```dart
class CognitoUserAttributeKey extends UserAttributeKey {
  const CognitoUserAttributeKey._(this.key, {this.readOnly = false});

  /// The JSON key for this attribute.
  final String key;

  /// Whether this attribute can only be read from Cognito.
  final bool readOnly;

  // remaining fields
}
```

Reply:
I like `name`. I think it's worth it.

Reply:
Just fyi - To use `name` we do have to update the static fields to const constructors since we have a static field right now called `name`

This:

```dart
static const name = CognitoUserAttributeKey._('name');
```

Needs to be:

```dart
const CognitoUserAttributeKey.name()  : name = 'name', readOnly = false;
```

The update is easy to make. There could be some confusion between the field `name` and constructor `name` but I think it make sense.

Reply:
Discussed with @dnys1 offline. `UserAttributeKey` / `CognitoUserAttributeKey` has a `.toString()` method, which is what we should push customers to use instead of `.key`. I updated the sample (including this location) to use `userAttributeKey.toString()`.